pkg/daemonset/daemonset.go

```diff
@@ -138,6 +138,71 @@ func (builder *Builder) WithNodeSelector(selector map[string]string) *Builder {
 	return builder
 }
 
+// WithHostNetwork applies HostNetwork to daemonset definition.
+func (builder *Builder) WithHostNetwork() *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Applying hostnetwork flag to daemonset %s in namespace %s",
```

Comment:
Applying or Enabling?

Reply:
This gets tricky on whats makes more sense.
Started off with "Enabling", had to modify to "Applying" after @kononovn review.